core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -51,34 +53,37 @@ export interface TopicChoice {
   id: string;
   topic: string;
 }
+export const PICK_FORMATS = {
+  parse: {dateInput: {month: 'short', year: 'numeric', day: 'numeric'}},
```

Comment:
Should `{dateInput: {month: 'short', year: 'numeric', day: 'numeric'}}` be an interface itself? Ditto elsewhere.

Reply:
Hello @chris7716, here are you referring to an interface like:
`interface DateInputFormat {
  month?: 'short' | 'long'; // Specify the month format (short or long)
  year?: 'numeric' | '2-digit'; // Specify the year format (numeric or 2-digit)
  day?: 'numeric' | '2-digit'; // Specify the day format (numeric or 2-digit)
}`

`export const PICK_FORMATS = {
  parse: {[dateInput: DateInputFormat]: {month: 'short', year: 'numeric', day: 'numeric'}},
  display: {
    dateInput: 'input',
    [monthYearLabel: DateInputFormat]: {year: 'numeric', month: 'short'},
    [dateA11yLabel: DateInputFormat]: {year: 'numeric', month: 'long', day: 'numeric'},
    [monthYearA11yLabel: DateInputFormat]: {year: 'numeric', month: 'long'},
  },
};`

Reply:
PTAL